angularjs-portal-home/src/main/webapp/partials/home-widget-view.html

```diff
@@ -25,9 +25,41 @@
     <div data-loading class='loading-gif'><img src="img/ajax-loader.gif" alt="Loading content, please wait"></div>
     
     <!-- Widget View -->
-    <div style="text-align:center; margin-top:30px">
-      <h3>Coming Soon</h3>
-      <p>Widgets are coming soon! Woohoo!</p>
-    </div>
+    <ul ui-sortable="sortableOptions" ng-model="layout" class="tile-list">
+      <!-- Widget Example -->
+      <li class="col-xs-12 col-sm-6 col-md-4 col-lg-3 no-padding list-container">  
```

Comment:
If you were feeling ambisous you could have a demo ribbon like [here](https://github.com/UW-Madison-DoIT/my-profile/blob/master/my-profile-webapp/src/main/webapp/partials/basic-info.html#L2)